docs/token_transferring/neonpass_usage.md

```diff
@@ -47,53 +47,59 @@ The following is an example of transferring NEON tokens from Neon EVM to Solana
 
 -->
 
-## Transfer tokens with NeonPass
-
-### Step 1: Set up wallets
+## Set up wallets
 
-1.1 Navigate to [NeonPass](https://neonpass.live/).
+1. Navigate to [NeonPass](https://neonpass.live/).
```

Comment:
@sukanyaparashar I see that you are suggesting a change in style >> as per this reworking of the Information Management structure of the walkthrough.

I am very happy for you to motivate for a budget to do some basic research and collect some evidence to back your assumption that one system is superior to others. For now though, please revert these.

Reply:
@m4sterbunny , I didn't actually suggested a style change. While adding the steps involved in transferring tokens from Solana to Neon EVM, I thought of making "Set up wallets" and "Connect wallets to NeonPass" common for both the tutorials. So, I did the numbering of the points in that way. Let me know the correct way to number the points.

Reply:
Main branch shows this structure: 
<img width="975" alt="image" src="https://github.com/neonlabsorg/neon-evm.docs/assets/20266893/84e50003-0e0c-4437-8d57-75897d225ff1">

Reply:
The tutorial is broken down into 3 steps and those steps are named according to the outcome
Sub-steps are then numbered so that by following the steps, the substep, the user achieves this outcome

Reply:
Please can you align this PR with the existing style -- thanks so much!

Reply:
Basically, my intention was to make two separate tutorials for using NeonPass -
1. Transfer tokens from Neon EVM to Solana (which is already there).
2. Transfer tokens from Solana to Neon EVM.
So, for both the cases, "Set up wallets" and "Connect wallet to NeonPass" will be repetitive, as this is the common step for both the tutorials. Hence, I separated those two out of the main tutorial. 

@m4sterbunny , I will make some new improvements to this PR once the announcement about the gas fee goes live tomorrow. Thanks for looking into this!

Reply:
@sukanyaparashar So, if I understand correctly, by creating 2 tutorials you have duplicated the information structure -- yes? Therefore, to avoid this issue, it is probably better to return to a single instance of the flow and apply tabs for the alternatives Tab 1: NEON > Sol, Tab 2: Sol> NEON. If you need help applying tabs, let me know -- there is a boilerplate to copy/paste in the readme.

Reply:
Yeah! I think tabs will be a better approach for that. I will try applying tabs. Thanks, @m4sterbunny.